store/mockstore/unistore/cophandler/mpp.go

```diff
@@ -168,6 +168,14 @@ func (b *mppExecBuilder) buildIdxScan(pb *tipb.IndexScan) (*indexScanExec, error
 	if b.dagReq.GetCollectRangeCounts() {
 		prevVals = make([][]byte, numIdxCols)
 	}
+	var partitionIDMap map[int64]struct{}
+	isGlobal := pb.IsGlobal != nil && *pb.IsGlobal
```

Comment:
If we go with extending the tipb protocol, is `isGlobal` really needed? is not `PartitionIds` enough? I.e. if `PartitionIds` is nil or of zero length, it already has the same practical meaning as of `isGlobal`?

Reply:
I understand `isGlobal` cannot be removed. 
w.g., If a table has only a partition, and it's in reorg. The query goes global index but the PartitionIds is empty now.